clang/test/CodeGenSYCL/check-work-group-attributes-match.cpp

```diff
@@ -0,0 +1,31 @@
+// RUN: %clang_cc1 -fno-sycl-force-inline-kernel-lambda -fsycl-is-device -internal-isystem %S/Inputs -triple spir64-unknown-unknown -disable-llvm-passes -sycl-std=2020 -opaque-pointers -emit-llvm -o - %s | FileCheck %s
```

Comment:
Could you please add -no-opaque-pointers test as well?

Reply:
The test should be the same, so I have added another RUN line with `-no-opaque-pointers`. Is that what you had in mind?

Reply:
Thank you.